Pspcl/Pspcl.Web/Views/AddStock/AddStock.cshtml

```diff
@@ -1,6 +1,7 @@
 ﻿@{
     Layout = "~/Views/Shared/_TwoColumnLayout.cshtml";
 }
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
```

Comment:
Can't we move it on layout page, as the font-awesome package can be used on almost all the pages.